src/components/InfoAboutPerson.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InfoAboutPerson({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {person.name ? `My name is ${person.name}` : null}
+      </h2>
+
+      <p className="Person__age">
+        I am
+        {person.age || '0'}
+      </p>
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
          I am {person.age}
        </p>
      )}
```
Use conditional rendering.